src/test/scala/org/infinispan/spark/suites/StreamingFailOverSuite.scala

```diff
@@ -32,7 +32,7 @@ class StreamingFailOverSuite extends FunSuite with SparkStream with MultipleServ
 
    protected def getProperties = {
       val props = new Properties()
-      props.put("infinispan.client.hotrod.server_list", s"localhost:$getServerPort")
+      props.put("infinispan.client.hotrod.server_list", s"${Cluster.getServerList}")
```

Comment:
I'm not sure this is correct. Connector should be able to fail over even in case we don't provide complete list of servers - e.g. following scenario should work: we start with 2 ISPN servers, start streaming, add 3 more ISPN servers and later on kill first 2 server we started with.